src/main.cpp

```diff
@@ -59,6 +59,11 @@
 #   include <unistd.h> // getpid()
 #endif
 
+#if !(defined(TILES) || defined(_WIN32))
+#include "imtui/imtui-impl-ncurses.h"
+#include "imtui/imtui-impl-text.h"
```

Comment:
Instead of adding dependencies directly on the imtui lib, put these shutdown methods in cata_imgui and handle the #ifdef stuff there.

Reply:
I've written a new cataimgui::client class and moved all ImGui_* and ImTui_* calls into it. I hope this sufficiently addresses this.